### PR TITLE
[Drop-In UI] Added ability to show Info Panel in Free Drive mode.

### PR DESCRIPTION
**Pull request by tomaszrybakiewicz**
Closes https://github.com/mapbox/mapbox-navigation-android/issues/5980

### Description

Changes: 

- Introduced `NavigationViewOptions.showInfoPanelInFreeDrive` that allows InfoPanel to be shown in FreeDrive state
- Fixed bottom sheet initial, hidden state layout location.
- QA Test App updates: Moved NavigationView customization to Drawer layout

### Screenshots or Gifs

_(Recoding of the QA Test App "Customized NavigationView" Activity captured on Pixel 6)_

https://user-images.githubusercontent.com/2678039/177371311-29ba6ef1-f688-4664-9f5f-aa0569027304.mp4

### CODE REVIEW

File: libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/coordinator/InfoPanelCoordinator.kt [hunk 1]
```diff
@@ -90,4 +104,30 @@ internal class InfoPanelCoordinator(
             setGuidelinePosition(bottomSheet)
         }
     }
+
+    /**
+     * An OnLayoutChangeListener that ensures the bottom sheet is always laid out at the bottom of
+     * the parent view when in STATE_HIDDEN.
+     */
```
Comment (tomaszrybakiewicz) [inline libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/coordinator/InfoPanelCoordinator.kt:111]: This `OnLayoutChangeListener` fixes the wrong bottom sheet initial position that causes this strange animation:

https://user-images.githubusercontent.com/2678039/177374383-724d1b4f-f079-400e-a31c-1caf509e7610.mp4

File: libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/ViewOptionsCustomization.kt [hunk 2]
```diff
@@ -39,6 +39,12 @@ class ViewOptionsCustomization {
      */
     var routeArrowOptions: RouteArrowOptions? = null
 
+    /**
+     * Set if the Info Panel should be visible for the Free Drive state.
+     * Set to `false` for the default behavior.
+     */
```
Comment (abhishek1508) [inline libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/ViewOptionsCustomization.kt:45]:
```suggestion
    /**
     * Set if the Info Panel should be visible for the Free Drive state. When set to `true`
     * [InfoPanelBinder] will only render an empty info panel. You are responsible to inflate 
     * appropriate [Binder] to render the content you would like to show on the info panel during 
     * free drive.
     * Set to `false` for the default behavior.
```

File: qa-test-app/src/main/res/drawable/ic_baseline_menu_24.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#000000"
```
Comment (abhishek1508) [inline qa-test-app/src/main/res/drawable/ic_baseline_menu_24.xml:1]: The menu icon is almost not visible in night mode. Replace `android:tine"#000000"` with
`android:tint="@color/colorOnSurface"`

**Review by abhishek1508 — COMMENTED** [review]
Thanks for adding this `Drawer` 💪🏻 . I was planning to cut a ticket to add it and pick up next week.

cc @tomaszrybakiewicz

**Comment by tomaszrybakiewicz** [comment]
@abhishek1508 Addressed all your feedback. I think it is ready to go. Please let me know if anything else needs to change.

**Pull request merged**